nntrainer/src/bn_layer.cpp

```diff
@@ -63,10 +64,9 @@ int BatchNormalizationLayer::setProperty(std::vector<std::string> values) {
     NN_RETURN_STATUS();
 
     unsigned int type = parseLayerProperty(key);
-
     switch (static_cast<PropertyType>(type)) {
     case PropertyType::input_shape:
-      status = dim.setTensorDim(values[0].c_str());
+      status = input_dim.setTensorDim(value.c_str());
       break;
```

Comment:
Is it necessary to give option about input shape? I wonder that is it possible to exist bn layer alone? I mean the input shape is totally depends on previous layer?

Reply:
#262 will enable input_shape option to bn layer anyway. So, we need to explicitly prohibit setting `input_shape` after all. I can't find any reason to prohibit setting input_shape for bnlayer. You are right that this is not necessary for bn layer though.